services/server/src/server/services/storageServices/AbstractDatabaseService.ts

```diff
@@ -98,6 +98,7 @@ export default abstract class AbstractDatabaseService {
       userdoc: compilerOutput?.userdoc || {},
       devdoc: compilerOutput?.devdoc || {},
       storageLayout: compilerOutput?.storageLayout || {},
+      sources: recompiledContract.compilerOutput?.sources || {},
```

Comment:
Where is this used?

Reply:
It's an information stored in the database. See the comment above. It's part of blockscout's tests.